### PR TITLE
🌐 Add French translation for Tutorial - Background Tasks

### PR DESCRIPTION
**Pull request by Smlep**
Hey everyone!

Here is the PR to translate the Background Tasks section into french.
See the french translation tracking issue [here](https://github.com/tiangolo/fastapi/issues/1972).

Thanks for the reviews👌

### CODE REVIEW

**Review by rjNemo — APPROVED** [review]

**Review by Jefidev — APPROVED** [review]
Looks good to me :smile: thanks for the work

File: docs/fr/docs/tutorial/background-tasks.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+L'opération d'écriture n'utilisant ni `async` ni `await`, on définit la fonction avec un `def` normal.
+
+```Python hl_lines="6-9"
+{!../../../docs_src/background_tasks/tutorial001.py!}
+```
+
+## Ajouter une tâche d'arrière-plan
+
+Dans votre *fonction de chemin*, passer votre fonction de tâche à l'objet de type `BackgroundTasks` (`background_tasks` ici) grâce à la méthode `.add_task()` :
```
Comment (NastasiaSaby) [inline docs/fr/docs/tutorial/background-tasks.md:41]: Shouldn't be "passez" in this case?

File: docs/fr/docs/tutorial/background-tasks.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+`.add_task()` reçoit comme arguments :
+
+* Une fonction de tâche à exécuter en arrière-plan (`write_notification`).
+* Les arguments positionnels à passer à la fonction de tâche dans l'ordre (`email`).
+* Les arguments nommés à passer à la fonction de tâche (`message="some notification"`).
+
+## Injection de dépendances
+
+Utiliser `BackgroundTasks` fonctionne aussi avec le système d'injection de dépendances, vous pouvez déclarer un paramètre de type `BackgroundTasks` à différents niveaux : dans une *fonction de chemin*, dans une dépendance, dans une sous-dépendance...
```
Comment (NastasiaSaby) [inline docs/fr/docs/tutorial/background-tasks.md:56]: Shouldn't we make two sentences of this one: Utiliser [...] dépendances. Vous pouvez déclarer...

**Review by NastasiaSaby — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thank you @Smlep ! :rocket: :cake: 

And thanks for the reviews @NastasiaSaby, @Jefidev, @rjNemo! :bow: :coffee:

**Pull request merged**